mock/py/mockbuild/plugins/lvm_root.py

```diff
@@ -382,6 +382,21 @@ def hook_remove_snapshot(self, name):
         lvm_do(['lvremove', '-f', self.vg_name + '/' + lv_name])
         self.buildroot.root_log.info("deleted {name} snapshot".format(name=name))
 
+    def hook_postupdate(self):
+        name = self.get_current_snapshot()
+        if re.match(r".*_updated_[\-\_0-9]{16}$", name):
```

Comment:
I fail to see the reason for this condition, can you please explain it in comment?

Reply:
You are right, the if is unnecessary here, the `re.sub` does the same thing without it.

Reply:
I still don't get what is the `sub` for :-) documentation is needed.

Reply:
Its for the situation when the current snapshot is already updated, so that there are no `postinit_updated_<date1>_updated_<date2>...` snapshots.